scripts/Publish-NginxPlus-Acr.ps1

```diff
@@ -128,20 +124,17 @@ function Publish-Image {
 
         [string]$acrLoginCommand = "az acr login --name $AcrName"
         Write-Host $acrLoginCommand
-        [string]$acrLoginOutput = Invoke-CommandLine -Command $acrLoginCommand
-        Write-Debug $acrLoginOutput
+        Invoke-CommandLine -Command $acrLoginCommand | Out-Null
 
         [string]$publishingTarget = '{0}.azurecr.io/{1}:{2}' -f $AcrName, "image/nginx-plus-ingress", $NGINXVersion
         [string]$dockerTagCommand = "docker tag private-registry.nginx.com/nginx-ic/nginx-plus-ingress:$NGINXVersion $publishingTarget"
         Write-Host $dockerTagCommand
-        [string]$dokerTagOutput = Invoke-CommandLine -Command $dockerTagCommand
-        Write-Debug $dokerTagOutput
```

Comment:
the removed debug statements, are we losing benefits here by removing them?

Reply:
No, because the debug statement is already there in the common function.. these statements were printing the same messages again.